toto.py

```diff
@@ -0,0 +1 @@
+coucou
```

Comment:
'coucou' should be 'hello'
```suggestion
'hello'
```